packages/project-types/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export enum FileName {
+    Manifest = 'manifest.json',
+    Package = 'package.json',
+    Ui5Yaml = 'ui5.yaml',
+    Ui5LocalYaml = 'ui5-local.yaml'
```

Comment:
`ui5-mock.yaml`?

Reply:
Added the ones that are used only and plan to extend them as we go.

Reply:
The `odata-service-writer` and `ui5-test-writer` check for `ui5-mock.yaml`, so we have a use case here.

Reply:
added it